pkg/rsyslog/setup-rsyslog.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
```

Comment:
Perhaps we should rename this script into build-rsyslog.sh to avoid confusion

Reply:
renamed the script to build-rsyslog.sh